### PR TITLE
✨Add backpack expander use feature

### PR DESCRIPTION
**Pull request by aethezxd**

### CODE REVIEW

File: app/handler/commands.js [hunk 1]
```diff
@@ -862,6 +863,23 @@ exports.handleMessage = function (steamID, message) {
...
+            client.chatMessage(steamID, 'I couldn\'t find any backpack expander(s) in your inventory.');
+            return;
+        }
+
+        crafting.useItem(assetids[0], function (err) {
+            if (err) {
+                client.chatMessage(steamID, 'Error occured while trying to use a backpack expander.');
+                return;
+            }
+            client.chatMessage(steamID, 'Inventory space increased by 100.');
```
Comment (Nicklason) [inline app/handler/commands.js]: Missing newline. Also log before sending message.

File: app/handler/commands.js [hunk 2]
```diff
@@ -862,6 +863,23 @@ exports.handleMessage = function (steamID, message) {
 
             client.chatMessage(steamID, 'Successfully uploaded new avatar.');
         });
+    } else if (isAdmin && command === 'expand') {
+        const assetids = [].concat(inventory.findBySKU('5050;6', false)).concat(inventory.findBySKU('5050;6;uncraftable', false));
+
+        if (assetids.length < 1) {
```
Comment (Nicklason) [inline app/handler/commands.js]: Just do assetids.length === 0

File: app/handler/commands.js [hunk 3]
```diff
@@ -862,6 +863,23 @@ exports.handleMessage = function (steamID, message) {
...
+            // No backpack expanders in your inventory.
+            client.chatMessage(steamID, 'I couldn\'t find any backpack expander(s) in your inventory.');
+            return;
+        }
+
+        crafting.useItem(assetids[0], function (err) {
+            if (err) {
+                client.chatMessage(steamID, 'Error occured while trying to use a backpack expander.');
```
Comment (Nicklason) [inline app/handler/commands.js]: Log the error and add the error message to the message you are sending

**Comment by Nicklason** [comment]
You also need to add the command to the !help response

File: app/handler/commands.js [hunk 4]
```diff
@@ -866,7 +866,7 @@ exports.handleMessage = function (steamID, message) {
     } else if (isAdmin && command === 'expand') {
         const assetids = [].concat(inventory.findBySKU('5050;6', false)).concat(inventory.findBySKU('5050;6;uncraftable', false));
 
-        if (assetids.length < 1) {
+        if (assetids.length === 1) {
```
Comment (offish) [inline app/handler/commands.js]: Change to 0

**Comment by aethezxd** [comment]
Sorry for the commits, I was too hasty.

**Pull request merged**